### ISSUE TITLE
Oscillating pitch at Cruise

### ISSUE THREAD

**Issue by Exp232** [Dros0phila/C208-MSFS2020-Fix#15]
The C208B oscillates pitch by about a half degree constantly while at cruise with AP ALT on, may be exacerbated by a heavy a/c. I have reason to believe the AP is not connected to the elevator trim tabs and, rather, is connected to the entire elevator, making fine corrections difficult for the AP. Can someone confirm this in the code?

**Comment by Bahnzo** [comment]
I think this is a problem across MSFS and not just this plane. I get plenty of AP oscillations with other planes as well.

**Comment by Exp232** [comment]
True, the AP is not perfect anywhere, but for some reason the trim level is not being affected by the AP in the C208, at least it's not showing that it is in game. Other a/c I've flown to show the pitch setting, i.e. the trim wheel is active, the trim tabs are at an angle, the trim UI in External View shows correctly. Non of this happens in the C208.

**Comment by Exp232** [comment]
Looks like the AP locks the trim at Neutral and then adjusts pitch using the entire elevator.

**Comment by Bahnzo** [comment]
I'm just not convinced this is something a mod of the plane can fix. Maybe it can be? I have noticed in the 2-3 flights in this plane so far that the AP is awful and absolutely pitches up and down with it's set initially.

**Comment by Dros0phila** [comment]
Yes, I also noticed that and the autopilot does seem to be connected to the elevator rather than the elevator trim surface. When you disconnect the AP you can sometimes noticed an abrupt change in the attitude, which shouldn't happen if the AP was linked to the trim surfaces. I don't think it's something that can be solved with just the mod thought, as it seems to be a problem of the simulator, but I'll take a look anyways as soon as I get back home.
